pkg/commands/user.go

```diff
@@ -47,43 +46,15 @@ func (r *UserCommand) ExecuteCommand(config *v1.Config, buildArgs *dockerfile.Bu
 		}
 	}
 
-	// Lookup by username
-	userObj, err := user.Lookup(userStr)
+	_, _, err = util.GetUserFromUsername(userStr, groupStr)
```

Comment:
Little bit confused here.. 
Not sure if its pertaining to this PR. 
But we are ignoring the resolved user and group and returning the input userStr:grpStr correct?
Is this command to just see if user and group can be resolved?